Tacta.EventStore/Tacta.EventStore.csproj

```diff
@@ -12,10 +12,11 @@
     <PackageIcon></PackageIcon>
     <PackageProjectUrl>https://tacta.io/</PackageProjectUrl>
     <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
-    <Version>1.6.4</Version>
-    <PackageVersion>1.6.4</PackageVersion>
+    <Version>1.6.5</Version>
```

Comment:
this change seems like a major breaking change and should be reflected in versioning? eg. it should be version 2.0.0 ?

Reply:
I agree, maybe we can combine it with (https://github.com/tacta-io/Tacta.EventStore/pull/27), so we avoid making two major versions one after another?